lib/python/tests/test_model.py

```diff
@@ -53,6 +53,14 @@ def test_create_get_from_id_and_update(
     assert model.model_id == get_model.model_id
 
 
+@pytest.mark.integration
+def test_search_models(integration_client):
+    models = Model.search(client=integration_client)
+
+    for model in models:
```

Comment:
A better way of doing this is `assert all(isinstance(model, Model) for model in models)` to avoid multiple assertions for the same test

Reply:
Done